_posts/en/meetings/2018-07-05-meeting.md

```diff
@@ -0,0 +1,232 @@
+
+---
+
+Topics discussed during this weekly meeting included what pull requests
+members of the project would like reviewers to focus on during the
+upcoming week (especially in light of the upcoming feature freeze for
+Bitcoin Core 0.17), alternating the time of the weekly meeting, and
+reducing the default minimum relay fee.
+
+## High priority for review
+
+**Background:** each meeting, Bitcoin Core developers discuss which Pull
+Requests (PRs) the meeting participants think most need review in the
+upcoming week.  Some of these PRs are related to code that contributors
+especially want to see in the next release; others are PRs that are
+blocking further work or which require significant maintenance (rebasing)
+to keep in a pending state.  Any capable reviewers are encouraged to
+visit the project's list of [current high-priority
+PRs][].
+
+**Discussion ([log][log hipri]):** Wladimir van der Laan stared the
+discussion by saying, "looks like there is only one thing left:
+[#12196][]," which adds a `scantxoutset` RPC method.  He then added,
+"reminder that the [0.17 feature freeze][#12624] is 2018-07-16, so in
+roughly a week."  After that date, contributors are discouraged from
+proposing PRs containing new features for the upcoming 0.17 release so
+that everyone can focus on finding and eliminating any bugs or other
+misbehaviors prior to release.
+
+With that deadline approaching, meeting participants suggested the
+following PRs be added to the high priority list:
+
+- **[#13547][]:** Make `signrawtransaction` give an error when amount is
+  needed but missing.  Suggested by AJ Towns and supported by Pieter
+  Wuille.
+
+- **[#12458][]:** Enforce that amount is provided for `signrawtransaction` prevtxs.
+  Suggested by Towns and supported by Wuille.
+
+- **[#13072][]:** Update `createmultisig` RPC to support segwit .
+  Suggested by Towns and supported by Wuille.
+
+- **[#11658][]:** During IBD, when doing pruning, prune 10% extra to
+  avoid pruning again soon after.  Suggested by Sjors Provoost.
+
+- **[#13557][]:** [BIP174][] PSBT serializations and RPCs.  Requested or
+  supported by Van der Laan, Wuille, Andrew Chow, and Gregory Maxwell.
+
+- **[#13298][]:** Net: Random delays *per network group* to obfuscate
+  transaction time.  Suggested by Gleb Naumenko, clearly supported by
+  Maxwell, and possibly supported by Wuille.  
+
+- **[#13414][]:** Support Gitlab API in github-merge.py.  Requested by
+  r-f, but mentioned as probably not relevant to Bitcoin Core 0.17 by
+  Van der Laan.
+
+**Conclusion:** all PRs mentioned above except the last, #13414, were
+added to the high-priority list.
+
+## Alternating meeting time
+
+**Background:** the weekly Bitcoin Core meeting is held at the same time
+each week, Thursdays at 19:00 UTC.  Converted to local time by AJ Towns
+(using northern hemisphere daylight savings time), this corresponds to
+the local times:
+
+| UTC   | NYC   | LAX   | Sydney | Tokyo | India | Europe |
```

Comment:
nit: Europe has several time zones. Technically this should be CEST (Central European Summer Time)

Reply:
(or you may want to put Paris / Berlin / whatever since most of the others are cities)

Reply:
Pushed new commit and updated preview.  Changes:

- Europe → Paris
- India → Delhi

Now they're all cities except for "UTC", which I think is more recognizable as that than "Reykjavik"